### DISCUSSION TITLE
SeisSol/Training - Docker image: TPV13 example does not produce plastic strain output

### DISCUSSION THREAD

**Discussion by VvanderHeiden** [SeisSol/SeisSol#530] [Q&A] [answered]
Dear all,

While going through the SeisSol training and following the slides of the ChEESE training on SeisSol (https://fs.hlrs.de/projects/par/events/2021/CHEESE/lectures/2021_ChEESE_SeisSol_training.pdf), I came across the section (slide 86) where the plastic strain output is imported into paraview.

However, the docker image available on the SeisSol/Training repository does not produce any plastic strain output for the TPV13 benchmark.

I have pulled the docker image to my local machine by using:
docker pull alicegabriel/seissol-training

when running this docker image the tpv13/parameters.par file does not contain the following lines (see complete .par file below):
!                 |strain     |eta
iPlasticityMask = 0 0 0 0 0 0 1

Once the docker image is pulled to my machine the 'Docker Desktop' dashboard indicates that the image was created 5 months ago. However, the parameters.par file on the SeisSol/Training/tpv13/ repository was updates 2 months ago.

Could it be that I am still pulling an old version of the SeisSol/Training docker image?

_old parameter file:_
[parameters.txt](https://github.com/SeisSol/SeisSol/files/8472575/parameters.txt)

**Comment by Thomas-Ulrich** [comment, marked as answer]
It should be in a file with the name finishing with "-low.xdmf".
(plastic strain output has been recently ported to the high-order volume output (that is it is now written in a file finishing by .xdmf without 
 "-low"), but the docker image has not been updated.

**Reply by Thomas-Ulrich** [reply]
I've fixed it.
